ipatests/test_integration/test_acme.py

```diff
@@ -183,6 +184,7 @@ def install(cls, mh):
 
         tasks.install_client(cls.master, cls.clients[0])
         tasks.install_replica(cls.master, cls.replicas[0])
+        Firewall(cls.clients[0]).enable_services(["http"])
```

Comment:
I think if you put this into prepare_acme_client() instead it will cover all cases.